<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDialog</class>
 <widget class="QWidget" name="DownloadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="KTitleWidget" name="m_titleWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="_2"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QListView" name="m_listView">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="m_providerLabel">
         <property name="text">
          <string>Provider:</string>
         </property>
         <property name="buddy">
          <cstring>m_providerCombo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="m_providerCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="m_categoryLabel">
         <property name="text">
          <string>Category:</string>
         </property>
         <property name="buddy">
          <cstring>m_categoryCombo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="m_categoryCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="orderLabel">
         <property name="text">
          <string>Order by:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KComboBox" name="m_sortCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="newestRadio">
         <property name="text">
          <string>Newest</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="ratingRadio">
         <property name="text">
          <string>Rating</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="mostDownloadsRadio">
         <property name="text">
          <string>Most Downloads</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="installedRadio">
         <property name="text">
          <string>Installed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Search:</string>
         </property>
         <property name="buddy">
          <cstring>m_searchEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="KLineEdit" name="m_searchEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="clickMessage">
          <string>Enter search phrase here</string>
         </property>
         <property name="showClearButton" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="KPushButton" name="closeButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>-1</number>
     </property>
     <item>
      <widget class="KNS3::ProgressIndicator" name="progressIndicator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KNS3::ProgressIndicator</class>
   <extends>QFrame</extends>
   <header>ui/progressindicator.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
